src/components/streamSelect/StreamSelect.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from "react";
+import "./StreamSelect.css";
+
+class StreamSelect extends Component {
+  constructor(props) {
+    super(props);
+    this.customStreamID = "";
+    this.state = {
+      currentStream: "5ddb98f5e4b0937e6a4507f2",
+      streamIsCustom: false,
+    };
+  }
+  updateField(event) {
+    this.customStreamID = event.target.value;
+  }
+  setCustomStream() {
+    this.props.setStream(this.customStreamID);
+    this.setState({ currentStream: "custom", streamIsCustom: true });
+  }
+  selectStream(event) {
+    this.props.setStream(event.target.value);
+    this.setState({ currentStream: event.target.value, streamIsCustom: false });
+  }
+  render() {
+    return (
+      <div className="options-bar">
+        <div className="stream-select-wrap">
+          <div className="stream-select-dropdown">
+            <span>Select Stream: </span>
```

Comment:
How about just showing:
Stream: [Livery Demo] ID: [Livery Demo ID]

And then showing Stream: [Custom] when the user changes the ID to a custom value?

Also, we currently change the URL to reflect this selection and to enable you to provide links to people for viewing a specified stream.